flask_resty/pagination.py

```diff
@@ -449,6 +449,13 @@ def get_filter_clause(self, column_cursors):
         )
 
         column, asc, value = column_cursors[-1]
+
+        # SQL Alchemy won't let you > or < a boolean, so we convert
+        # to an integer, the DB's seem to handle this just fine
+        if isinstance(value, bool):
+            column = sa.cast(column, sa.Integer)
+            value = 1 if value else 0
```

Comment:
nitpick: This might not be necessary; bools are just ints under the hood, so I'd be somewhat surprised if `column > [True|False]` didn't just work.

If you do need an int, you can do

```suggestion
            value = int(value)
```